ios/ZooniverseMobile/Info.plist

```diff
@@ -17,7 +17,7 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>2.4.2</string>
+	<string>2.6.0</string>
```

Comment:
Are we on `2.6.1` now?

Reply:
As of the latest, yes. That gets updated by the script, so we have to set it one patch/minor lower than we're at in the code.

Reply:
I've been handling this by putting it one lower in the code, then running the script, then updating the code and pushing.